outreach/podcast/ideas/structure.md

```diff
@@ -0,0 +1,34 @@
+# OpCode Podcast Structure Ideas*
+*Progress over Perfection
+
+## Overview of Show:
+
+1. Message by David Molina
+2. Welcome and Intro
+3. Table of Contents
```

Comment:
What would this mean exactly from the audio perspective?

Reply:
Similar function - set up for the listener to know what to expect. You could probably consider that a regular "intro" but in this list I referenced `Welcome and Intro` as the actual music and the "Hello my name is" part of the podcast